### DISCUSSION TITLE
post-processing tools for Molecular Dynamics

### DISCUSSION THREAD

**Discussion by LiuXiaohui123321** [deepmodeling/abacus-develop#1600] [General]
Are there any tools to post-process the MD data of ABACUS? For example, how to get the MD trajectories, MSD, VACF?

**Comment by dyzheng** [comment]
As far as I know, maybe @Liu-RX or @YuLiu98 can help with MD trahectories and MSD analysis.
I am sorry that I know nothing about VACF.
I suggest you can calculate RDF(radial distribution function) to analyze the MD status.

**Comment by mohanchen** [comment]
We have developed a package that can do the post-processing of data. Here is the link: https://github.com/MCresearch/Candela

**Comment by mohanchen** [comment]
see above
